src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
It's important to have an ErrorBoundary for your app. Otherwise if an error occurs React will unmount and show a white screen. See storybook to see how this looks. You'll wanna tweak this to look nicer.

You can also add nested ErrorBoundaries for portions of the app so that the entire app doesn't crash if a portion fails.

Reply:
Added verbatim